src/libraries/restore/runtime/runtime.depproj

```diff
@@ -97,8 +101,8 @@
   <Target Name="OverrideRuntimeMono"
           DependsOnTargets="ResolveRuntimeFilesFromLocalBuild"
           AfterTargets="AfterResolveReferences;FilterNugetPackages"
-          Condition="'$(RuntimeFlavor)' == 'Mono'">
-    <ItemGroup Condition="'$(BinPlaceTestRuntimePack)' != 'true'">
+          Condition="'$(TargetsMobile)' != 'true'">
```

Comment:
This target condition has to be: `'$(RuntimeFlavor)' == 'Mono' and '$(TargetsMobile)' != 'true'`, right?

Reply:
Correct